modules/candidate_profile/test/TestPlan.md

```diff
@@ -50,6 +50,7 @@ that widget (ie. the media module for CandID 587630 (DCC090) or CandID 300001 (M
 4. Ensure that, when the module which added the extra `CandidateInfo` terms
    is disabled, the terms from that module no longer show up in the
    `Candidate Info` card.
+5. Ensure that all the instrument links work without 500 error.
```

Comment:
Hi @kongtiaowang, this test case is already existing in the test plan found here: https://github.com/aces/Loris/blob/04267e4b765d40480e607bd54392266379a69733/modules/instruments/test/TestPlan.md#visit-the-candidate-dashboard-module

See, "Candidate Widget - "Behavioural Data" for Candidate instruments entry." and maybe it should be reworded to be similar to what you wrote here.